app/src/main/java/org/oppia/android/app/options/LanguageItemViewModel.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.android.app.options
+
+import org.oppia.android.app.viewmodel.ObservableViewModel
+
+/** Language item view model for the recycler view in [AppLanguageFragment] and [AudioLanguageFragment]. */
```

Comment:
Make this multiline as it is going out-of-bounds.
```
/** 
 * Language item view model for the recycler view in [AppLanguageFragment] and 
 * [AudioLanguageFragment].
 */
```

Reply:
Done